src/BundleAdjustmentNLSFunctions.jl

```diff
@@ -176,7 +176,7 @@ end
 projection!(x, c, r2, v, P1) =
   projection!(x, view(c, 1:3), view(c, 4:6), c[7], c[8], c[9], r2, v, P1)
 
-function scaling_factor(point::AbstractVector, k1, k2)
+function scaling_factor(point, k1, k2)
```

Comment:
did you mean to undo this?

Reply:
Yes, it was to avoid having to rebase since it is the modification in pull request #62. But I guess it is more confusing to do it like this. I will change it back.